### ISSUE TITLE
[Bug]: [Bulk load] Milvus crashes when importing string primary key data files

### ISSUE THREAD

**Issue by yanliang567** [milvus-io/milvus#16858]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:master-20220507-c7935d1a
- Deployment mode(standalone or cluster): standalone
- SDK version(e.g. pymilvus v2.0.0rc2): 2.1.0.dev49
```

### Current Behavior

Milvus crashes when importing row based data file with string as primary key
[bulkload.strpk.crash.2.log](https://github.com/milvus-io/milvus/files/8650509/bulkload.strpk.crash.2.log)

### Expected Behavior

import data successfully

### Steps To Reproduce

```markdown
1. prepare row based data file with string as primary key
2. create collection
3. bulk load the data file
```

### Anything else?

row_cust_str_pk_float_vectors_multi_scalars_4d_10.json:
```
{
"rows":[
{"uid":"Dm4aWrbNzhmjwCTEnCJ9LDPO2N09sqysxgVfbH9Zmn3nBzmwsmk0eZN6x7wSAoPQ","int_scalar":9070353,"float_scalar":0.9798043638085004,"string_scalar":"ShQ44OX0z8kGpRPhaXmfSsdH7JHq5DsZzu0e2umS1hrWG0uONH2RIIAdOECaaXir","bool_scalar":true,"vectors":[0.5040062902126952, 0.8297619818664708, 0.20248342801564806, 0.12834786423659314]},
{"uid":"RP50U0d2napRjXu94a8oGikWgklvVsXFurp8RR4tHGw7N0gk1b7opm59k3FCpyPb","int_scalar":8505288,"float_scalar":0.937913432198687,"string_scalar":"Ld4b0avxathBdNvCrtm3QsWO1pYktUVR7WgAtrtozIwrA8vpeactNhJ85CFGQnK5","bool_scalar":false,"vectors":[0.528232122836893, 0.6916116750653186, 0.41443762522548705, 0.26624344144792056]},
{"uid":"oxhFkQitWPPw0Bjmj7UQcn4iwvS0CU7RLAC81uQFFQjWtOdiB329CPyWkfGSeYfE","int_scalar":4392660,"float_scalar":0.32381232630490264,"string_scalar":"EmAlB0xdQcxeBtwlZJQnLgKodiuRinynoQtg0eXrjkq24dQohzSm7Bx3zquHd3kO","bool_scalar":false,"vectors":[0.7978693027281338, 0.12394906726785092, 0.42431962903815285, 0.4098707807351914]},
{"uid":"sxoEL4Mpk1LdsyXhbNm059UWJ3CvxURLCQczaVI5xtBD4QcVWTDFUW7dBdye6nbn","int_scalar":7927425,"float_scalar":0.31074026464844895,"string_scalar":"fdY2beCvs1wSws0Gb9ySD92xwfEfJpX5DQgsWoISylBAoYOcXpRaqIJoXYS4g269","bool_scalar":true,"vectors":[0.3716157812069954, 0.006981281113265229, 0.9007003458552365, 0.22492634316191004]},
{"uid":"g33Rqq2UQSHPRHw5FvuXxf5uGEhIAetxE6UuXXCJj0hafG8WuJr1ueZftsySCqAd","int_scalar":9288807,"float_scalar":0.4953578200336135,"string_scalar":"6f8Iv1zQAGksj5XxMbbI5evTrYrB8fSFQ58jl0oU7Z4BpA81VsD2tlWqkhfoBNa7","bool_scalar":false,"vectors":[0.5921374209648096, 0.04234832587925662, 0.7803878096531548, 0.1964045837884633]},
{"uid":"ACIJd7lTXkRgUNmlQk6AbnWIKEEV8Z6OS3vDcm0w9psmt9sH3z1JLg1fNVCqiX3d","int_scalar":1173595,"float_scalar":0.9000745450802002,"string_scalar":"gpj9YctF2ig1l1APkvRzHbVE8PZVKRbk7nvW73qS2uQbY5l7MeIeTPwRBjasbY8z","bool_scalar":true,"vectors":[0.4655121736168688, 0.6195496905333787, 0.5316616196326639, 0.3417492053890768]},
{"uid":"f0wRVZZ9u1bEKrAjLeZj3oliEnUjBiUl6TiermeczceBmGe6M2RHONgz3qEogrd5","int_scalar":3722368,"float_scalar":0.7212299175768438,"string_scalar":"xydiejGUlvS49BfBuy1EuYRKt3v2oKwC6pqy7Ga4dGWn3BnQigV4XAGawixDAGHN","bool_scalar":false,"vectors":[0.6173164237304075, 0.374107748459483, 0.3686321416317251, 0.585725336391797]},
{"uid":"uXq9q96vUqnDebcUISFkRFT27OjD89DWhok6urXIjTuLzaSWnCVTJkrJXxFctSg0","int_scalar":1940731,"float_scalar":0.9524404085944204,"string_scalar":"ZXSNzR5V3t62fjop7b7DHK56ByAF0INYwycKsu6OxGP4p2j0Obs6l0NUqukypGXd","bool_scalar":false,"vectors":[0.07178869784465443, 0.4208459174227864, 0.5882811425075762, 0.6867753592116734]},
{"uid":"EXDDklLvQIfeCJN8cES3b9mdCYDQVhq2iLj8WWA3TPtZ1SZ4Jpidj7OXJidSD7Wn","int_scalar":2158426,"float_scalar":0.23770219927963454,"string_scalar":"9TNeKVSMqTP8Zxs90kaAcB7n6JbIcvFWInzi9JxZQgmYxD5xLYwaCoeUzRiNAxAg","bool_scalar":false,"vectors":[0.5659468293534021, 0.6275816433340369, 0.3978846871291008, 0.3571179679645908]},
{"uid":"mlaXOgYvB88WWRpXNyWv6UqpmvIHrC6pRo03AtaPLMpVymu0L9ioO8GWa1XgGyj0","int_scalar":198279,"float_scalar":0.020343767010139513,"string_scalar":"AblYGRZJiMAwDbMEkungG0yKTeuya4FgyliakWWqSOJ5TvQWB9Ki2WXbnvSsYIDF","bool_scalar":true,"vectors":[0.5374636140212398, 0.7655373567912009, 0.05491796821609715, 0.349384366747262]}
]
}

```

**Comment by yhmo** [comment]
Fixed by #16958

**Comment by yanliang567** [comment]
verified fixed on master-20220513-52fc9a55
